lib/node.js

```diff
@@ -1356,12 +1356,15 @@ declare module "fs" {
       ...
     }>,
     readdir(path: FSPromisePath, options?: string | EncodingOptions): Promise<string[]>,
-    readFile(path: FSPromisePath | FileHandle, options: string): Promise<string>,
-    readFile(path: FSPromisePath | FileHandle, options?: EncodingFlag): Promise<Buffer>,
-    readlink(path: FSPromisePath, options: BufferEncoding): Promise<Buffer>,
-    readlink(path: FSPromisePath, options?: string | EncodingOptions): Promise<string>,
-    realpath(path: FSPromisePath, options: BufferEncoding): Promise<Buffer>,
-    realpath(path: FSPromisePath, options?: string | EncodingOptions): Promise<string>,
+    readFile:
+      & (path: FSPromisePath | FileHandle, options: string) => Promise<string>
```

Comment:
Have you tested this works? From my experience migrating lodash, this would mean

`(path: FSPromisePath | FileHandle, options: string)` return  `Promise<string> & ((path: FSPromisePath | FileHandle, options?: EncodingFlag) => Promise<Buffer>)`

You may want to change this to format like this
```
& ((path: FSPromisePath | FileHandle, options: string) => Promise<string>)
& ((path: FSPromisePath | FileHandle, options?: EncodingFlag) => Promise<Buffer>)
```

Reply:
oh, absolutely. You're right

Reply:
fixed, thanks ;)